### PR TITLE
(chore) Supporting dbt sync without updating the db connection configuration on Superset

### PR DESCRIPTION
**Pull request by Vitor-Avila**
Currently, the CLI would always update the DB connection configuration on the Workspace using the information from the `profiles.yml` file (regardless if `--import-db` was passed or not). The flag was only validated in case **no database connection was found**.

This PR introduces the ability to perform a dbt sync without updating the DB connection. This is very useful in case the dbt configuration uses different credentials, or even a different authentication method that might not be supported by Superset.

### CODE REVIEW

File: src/preset_cli/cli/superset/sync/dbt/databases.py
Comment (Vitor-Avila) [inline src/preset_cli/cli/superset/sync/dbt/databases.py:81]: Previously, this would always be executed (which pretty much builds the `SQLAlchemy` URI from the connection settings on the `profiles.yml`). Now it's only executed in case `--import-db` is passed.

File: src/preset_cli/cli/superset/sync/dbt/databases.py
Comment (Vitor-Avila) [inline src/preset_cli/cli/superset/sync/dbt/databases.py:88]: In case `--import-db` is not passed, I added this to fetch the `SQLAlchemy` URI from the Superset API instead. The password wouldn't be included, but I don't think this would be a problem since we're not updating the DB connection.

**Comment by dweaver33** [comment]
I've covered this fix here:  https://github.com/preset-io/backend-sdk/pull/169

**Comment by Vitor-Avila** [comment]
Hey @dweaver33, 

Thank you so much for your interest on contributing to the CLI. I'm afraid that due to license restrictions, we're currently not accepting contributions from outside of Preset. I was actually going to add a comment to your PR/Issue once this gets merged.

Thank you so much for your understanding.

**Comment by dweaver33** [comment]
> Hey @dweaver33,
> 
> Thank you so much for your interest on contributing to the CLI. I'm afraid that due to license restrictions, we're currently not accepting contributions from outside of Preset. I was actually going to add a comment to your PR/Issue once this gets merged.
> 
> Thank you so much for your understanding.

Ha!  Didn't realize you were a maintainer here. 

In that case, can you also look at my fix in that same PR for the exposures?  #177

**Review by betodealmeida — APPROVED** [review]
This looks great! Thanks, Vitor!

**Comment by Vitor-Avila** [comment]
> > Hey @dweaver33,
> > Thank you so much for your interest on contributing to the CLI. I'm afraid that due to license restrictions, we're currently not accepting contributions from outside of Preset. I was actually going to add a comment to your PR/Issue once this gets merged.
> > Thank you so much for your understanding.
> 
> Ha! Didn't realize you were a maintainer here.
> 
> In that case, can you also look at my fix in that same PR for the exposures? #177

I'll take a look on that ticket, @dweaver33. Thank you so much!

**Pull request merged**
